bundles/binding/org.openhab.binding.mochadx10/src/main/java/org/openhab/binding/mochadx10/internal/MochadX10Binding.java

```diff
@@ -81,9 +232,73 @@ public void updated(Dictionary<String, ?> properties) throws ConfigurationExcept
 					throw new ConfigurationException(port, "The specified port \"" + port + "\" is not a valid port number.");
 				}
 			}
+			
+			initializeBinding();
```

Comment:
please issues a disconnect when this Binding is deactivated() (you'll find a proper callback method for that)